src/plugins/ucc_discovery_plugin.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "expression/abstract_expression.hpp"
+#include "logical_query_plan/abstract_lqp_node.hpp"
+#include "utils/abstract_plugin.hpp"
+#include "utils/pausable_loop_thread.hpp"
+
+namespace opossum {
+
+class UCCCandidate {
+ public:
+  UCCCandidate(std::string table_name, ColumnID column_id) : _table_name(table_name), _column_id(column_id) {}
+
+  const std::string table_name() const {
+    return this->_table_name;
```

Comment:
Please use `this` as little as possible as specified in the hyrise style guide
```suggestion
    return _table_name;
```